source/publish/quick_start/publish.rst

```diff
@@ -0,0 +1,30 @@
+===============
+Publish the map
+===============
+
+.. contents::
+   :depth: 3
+
+Reminder of Lizmap architecture
+===============================
+
+.. image:: /images/architecture.jpg
```

Comment:
This image is already in another page, that's why I have removed it
https://docs.lizmap.com/current/fr/introduction.html#lizmap-architecture